nearcore/src/lib.rs

```diff
@@ -232,6 +233,9 @@ pub fn start_with_config_and_synchronization(
         config.client_config.log_summary_period,
     )?;
 
+    // Initialize genesis STATE either from config or state/storage before other components.
+    initialize_genesis_state(storage.get_hot_store(), &config.genesis, Some(home_dir));
```

Comment:
As it changes behaviour of the whole node, can we run nayduck for the PR?

Reply:
https://nayduck.near.org/#/run/3140

Reply:
Looks like the tests are passing